docs/nl/docs/environment-variables.md

```diff
@@ -0,0 +1,298 @@
+
+// Maar als we eerst een omgevingsvariabele aanmaken
+$ export MY_NAME="Wade Wilson"
+
+// en het programma dan opnieuw aanroepen
+$ python main.py
+
+// kan het de omgevingsvariabele nu wel uitlezen
+
+Hello Wade Wilson from Python
+```
+
+</div>
+
+////
+
+//// tab | Windows PowerShell
+
+<div class="termy">
+
+```console
+// Hier stellen we de omgevingsvariabelen nog niet in
+$ python main.py
+
+// Omdat we de omgevingsvariabelen niet hebben ingesteld, krijgen we de standaardwaarde
+
+Hello World from Python
+
+// Maar als we eerst een omgevingsvariabele aanmaken
+$ $Env:MY_NAME = "Wade Wilson"
+
+// en het programma dan opnieuw aanroepen
+$ python main.py
+
+// kan het de omgevingsvariabele nu wel uitlezen
+
+Hello Wade Wilson from Python
+```
+
+</div>
+
+////
+
+Omdat omgevingsvariabelen buiten de code kunnen worden ingesteld, maar wel door de code kunnen worden gelezen en niet hoeven te worden opgeslagen (gecommit naar `git`) met de rest van de bestanden, worden ze vaak gebruikt voor configuraties of **instellingen**.
+
+Je kunt ook een omgevingsvariabele maken die alleen voor een **specifieke programmaaanroep** beschikbaar is, die alleen voor dat programma beschikbaar is en alleen voor de duur van dat programma.
+
+Om dat te doen, maak je het vlak voor het programma zelf aan, op dezelfde regel:
+
+<div class="termy">
+
+```console
+// Maak een omgevingsvariabele MY_NAME in de regel voor deze programma-aanroep
+$ MY_NAME="Wade Wilson" python main.py
+
+// Nu kan het de omgevingsvariabele lezen
+
+Hello Wade Wilson from Python
+
+// De omgevingsvariabelen bestaan daarna niet meer
+$ python main.py
+
+Hello World from Python
+```
+
+</div>
+
+/// tip
+
+Je kunt er meer over lezen op <a href="https://12factor.net/config" class="external-link" target="_blank">The Twelve-Factor App: Config</a>.
+
+///
+
+## Types en Validatie
+
+Deze omgevingsvariabelen kunnen alleen **tekstreeksen** verwerken, omdat ze extern zijn aan Python, compatibel moeten zijn met andere programma's en de rest van het systeem (zelfs met verschillende besturingssystemen, zoals Linux, Windows en macOS).
+
+Dat betekent dat **elke waarde** die in Python uit een omgevingsvariabele wordt gelezen **een `str` zal zijn** en dat elke conversie naar een ander type of elke validatie in de code moet worden uitgevoerd.
+
+Meer informatie over het gebruik van omgevingsvariabelen voor het verwerken van **applicatie instellingen** vind je in de [Geavanceerde gebruikershandleiding - Instellingen en Omgevingsvariabelen](./advanced/settings.md){.internal-link target=_blank}.
+
+## `PATH` Omgevingsvariabele
+
+Er is een **speciale** omgevingsvariabele met de naam **`PATH`**, die door de besturingssystemen (Linux, macOS, Windows) wordt gebruikt om programma's te vinden die uitgevoerd kunnen worden.
+
+De waarde van de variabele `PATH` is een lange tekstreeks die bestaat uit mappen die gescheiden worden door een dubbele punt `:` op Linux en macOS en door een puntkomma `;` op Windows.
```

Comment:
```suggestion
De waarde van de variabele `PATH` is een lange string die bestaat uit mappen die gescheiden worden door een dubbele punt `:` op Linux en macOS en door een puntkomma `;` op Windows.
```
I've never seen the usage of the word "tekstreeks" before, though that might be just me. Wondering whether we can just say "string" here.